deploy/operator.yml

```diff
@@ -12,275 +12,471 @@ parameters:
   - name: VALIDATION_PERCENTAGE_THRESHOLD
-    value: "5"
+    value: '5'
   - name: VALIDATION_PERCENTAGE_THRESHOLD_INIT
-    value: "5"
+    value: '5'
   - name: VALIDATION_ATTEMPTS_THRESHOLD
-    value: "10"
+    value: '10'
   - name: VALIDATION_ATTEMPTS_THRESHOLD_INIT
-    value: "60"
+    value: '60'
   - name: VALIDATION_PERIOD_MINUTES
-    value: "60"
+    value: '60'
   - name: VALIDATION_LAG_COMPENSATION_SECONDS
-    value: "600"
+    value: '600'
   - name: CONNECT_CLUSTER
-    value: "xjoin-kafka-connect-strimzi"
+    value: xjoin-kafka-connect-strimzi
   - name: ES_CONNECTOR_TASKS_MAX
-    value: "50"
+    value: '50'
   - name: JENKINS_MANAGED_VERSION
-    value: "v1.0"
+    value: v1.0
   - name: KAFKA_TOPIC_PARTITIONS
-    value: "500"
+    value: '500'
   - name: KAFKA_TOPIC_REPLICAS
-    value: "2"
+    value: '2'
   - name: ES_CONNECTOR_CONFIG
-    value: '{
-      "tasks.max": "{{.ElasticSearchTasksMax}}",
-      "topics": "{{.Topic}}",
-      "key.ignore": "false",
-      "connection.url": "${env:ELASTICSEARCH_ENDPOINT}",
-      {{if .ElasticSearchUsername}}"connection.username": "{{.ElasticSearchUsername}}",{{end}}
-      {{if .ElasticSearchPassword}}"connection.password": "{{.ElasticSearchPassword}}",{{end}}
-      "type.name": "_doc",
-      "transforms": "valueToKey, extractKey, expandJSON, deleteIf, flattenList, flattenListString, renameTopic",
+    value: >-
+      { "tasks.max": "{{.ElasticSearchTasksMax}}", "topics": "{{.Topic}}",
+      "key.ignore": "false", "connection.url": "${env:ELASTICSEARCH_ENDPOINT}",
+      {{if .ElasticSearchUsername}}"connection.username":
+      "{{.ElasticSearchUsername}}",{{end}} {{if
+      .ElasticSearchPassword}}"connection.password":
+      "{{.ElasticSearchPassword}}",{{end}} "type.name": "_doc", "transforms":
+      "valueToKey, extractKey, expandJSON, deleteIf, flattenList,
+      flattenListString, renameTopic",
       "transforms.valueToKey.type":"org.apache.kafka.connect.transforms.ValueToKey",
       "transforms.valueToKey.fields":"id",
       "transforms.extractKey.type":"org.apache.kafka.connect.transforms.ExtractField$Key",
-      "transforms.extractKey.field":"id",
-      "transforms.expandJSON.type": "com.redhat.insights.expandjsonsmt.ExpandJSON$Value",
-      "transforms.expandJSON.sourceFields": "tags",
-      "transforms.deleteIf.type": "com.redhat.insights.deleteifsmt.DeleteIf$Value",
-      "transforms.deleteIf.field": "__deleted",
-      "transforms.deleteIf.value": "true",
-      "transforms.flattenList.type": "com.redhat.insights.flattenlistsmt.FlattenList$Value",
+      "transforms.extractKey.field":"id", "transforms.expandJSON.type":
+      "com.redhat.insights.expandjsonsmt.ExpandJSON$Value",
+      "transforms.expandJSON.sourceFields": "tags", "transforms.deleteIf.type":
+      "com.redhat.insights.deleteifsmt.DeleteIf$Value",
+      "transforms.deleteIf.field": "__deleted", "transforms.deleteIf.value":
+      "true", "transforms.flattenList.type":
```

Comment:
similar odd formatting here. Seems to sometimes create `key:value, key:` on a single line. Not a huge deal but kind of confusing.